snmp/datadog_checks/snmp/data/conf.yaml.example

```diff
@@ -342,3 +344,10 @@ instances:
     #    OID: 1.3.6.1.2.1.1.5
     #    symbol: sysName
     #    tag: snmp_host
+    #  - # With regular expression matching
+    #    MIB: SNMPv2-MIB
+    #    symbol: sysName
+    #    match: '(.*)-(.*)
```

Comment:
```suggestion
    #    match: (.*)-(.*)
```

Reply:
For some reasons I thought it broke yaml load, but it seems to work!